it/src/main/java/org/corfudb/universe/scenario/fixture/Fixtures.java

```diff
@@ -0,0 +1,107 @@
+package org.corfudb.universe.scenario.fixture;
+
+import com.google.common.collect.ImmutableList;
+import lombok.Builder;
+import lombok.Getter;
+import org.corfudb.universe.node.Node;
+import org.corfudb.universe.service.Service.ServiceParams;
+import org.slf4j.event.Level;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+import static lombok.Builder.Default;
+import static org.corfudb.universe.cluster.Cluster.ClusterParams;
+import static org.corfudb.universe.node.CorfuServer.Mode;
+import static org.corfudb.universe.node.CorfuServer.Persistence;
+import static org.corfudb.universe.node.CorfuServer.ServerParams;
+
+/**
+ * Fixture factory provides predefined fixtures
+ */
+public interface Fixtures {
+
+    @Builder
+    @Getter
+    class SingleServerFixture implements Fixture<ServerParams> {
+        @Default
+        private final int port = 9000;
+        @Default
+        private final Mode mode = Mode.SINGLE;
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)